### DISCUSSION TITLE
Send Another Request with Foreign Keys

### DISCUSSION THREAD

**Discussion by tonyhart7** [fastapi/fastapi#8720] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class Post(Base):
    __tablename__ = "posts"
    __table_args__ = {'extend_existing': True}
    
    id = Column(Integer, primary_key=True, nullable=False)
    title = Column(String, nullable=False)
    content = Column(String, nullable=False)
    published = Column(Boolean, server_default='TRUE', nullable=False)
    created_at = Column(TIMESTAMP(timezone=True),
                        nullable=False, server_default=text('now()'))
    owner_id = Column(Integer, ForeignKey(
        "users.id", ondelete="CASCADE"), nullable=False)

    owner = relationship("User")


class User(Base):
    __tablename__ = "users"
    __table_args__ = {'extend_existing': True}
    
    id = Column(Integer, primary_key=True, nullable=False)
    email = Column(String, nullable=False, unique=True)
    password = Column(String, nullable=False)
    created_at = Column(TIMESTAMP(timezone=True),
                        nullable=False, server_default=text('now()'))
```

### Description

so I want to send post request to the User with creator of user with foreign key 
as You can see, so 1 post with 1 user associated with it (just typical, you know)

but I cant do that and I want to use another way, Is there another way ?

### Operating System

Windows

### Operating System Details

Windows 11

### FastAPI Version

0.70.1

### Python Version

Python 3.10.0

**Comment by tonyhart7** [comment]
~~~
sqlalchemy.exc.InvalidRequestError: Multiple classes found for path "User" in the registry of this declarative base. Please use a fully module-qualified path.
~~~

for anyone wondering why I cant do that

**Comment by n8sty** [comment]
This doesn't seem to be a FastAPI specific issue and is probably better suited for Stackoverflow or a SQLAlchemy specific resource. If there's nothing FastAPI related, please close.
